spacy/cli/info.py

```diff
@@ -16,17 +17,18 @@ def info_cli(
     markdown: bool = Opt(False, "--markdown", "-md", help="Generate Markdown for GitHub issues"),
     silent: bool = Opt(False, "--silent", "-s", "-S", help="Don't print anything (just return)"),
     exclude: str = Opt("labels", "--exclude", "-e", help="Comma-separated keys to exclude from the print-out"),
+    url: bool = Opt(False, "--url", "-u", help="Print the pipeline download URL"),
```

Comment:
This is likely a matter of personal preference, so feel free to ignore.
```suggestion
    url: bool = Opt(False, "--url", "-u", help="Print the URL to download the pipeline from"),
```